app/com/linkedin/drelephant/exceptions/azkaban/AzkabanJobLogExceptionFingerprinting.java

```diff
@@ -0,0 +1,85 @@
+package com.linkedin.drelephant.exceptions.azkaban;
+
+import com.linkedin.drelephant.clients.WorkflowClient;
+import com.linkedin.drelephant.configurations.scheduler.SchedulerConfigurationData;
+import com.linkedin.drelephant.exceptions.util.ExceptionInfo;
+import com.linkedin.drelephant.util.InfoExtractor;
+import com.linkedin.drelephant.util.Utils;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.log4j.Logger;
+
+import static com.linkedin.drelephant.exceptions.util.ExceptionUtils.ConfigurationBuilder.*;
+import static com.linkedin.drelephant.exceptions.util.ExceptionUtils.*;
+import static controllers.api.v1.JsonKeys.*;
+
+
+public class AzkabanJobLogExceptionFingerprinting {
```

Comment:
The class's name is AzkabanExceptionLogAnalyzer.

For the issue of "extending ExceptionFingerprinting interface", we have discussed earlier that we cannot implement "**classifyException**" as that is not in the scope of TonYExceptionFingerprinting. **ExceptionFingerprinting** interface was designed for purpose of EF for AutoTuning, but this has changed since the rollout of Generalised EF for user jobs. Additionally, TonY is not supported by AutoTuning so cannot implement `classifyException` at all.

Reply:
ClassifyException is general method   . Ideally ExceptionFingerprinting  responsibility is to classify the exception (even for TonY case we need the classification of exceptions) .

Reply:
Have u changed class name , from AzkabanJobLogExceptionFingerprinting => AzkabanExceptionLogAnalyzer ?